### DISCUSSION TITLE
How to monitor plotmanager with prometheus

### DISCUSSION THREAD

**Discussion by peterMa2009** [swar/Swar-Chia-Plot-Manager#1129] [Q&A]
Who can teach me how to use prometheus to monitor the plot manager？thank you very much！

**Comment by realdevnullius** [comment]
I had the same question :) The best I got was a webpage with some numbers. Nothing fancy, at all! :( 

Any links would be appreciated too ;-)

**Reply by peterMa2009** [reply]
nobody can helpe us!

**Reply by realdevnullius** [reply]
but we want the pretty pictures! :(
